api/middleware/middleware.go

```diff
@@ -40,10 +40,10 @@ func getOrg(req *http.Request, multiTenant bool) (int, error) {
 		return 0, nil
 	}
 	org, err := strconv.Atoi(orgStr)
-	if err != nil {
+	if err != nil || org < 1 {
```

Comment:
now the public org id is configurable, right? so if somebody sets it to a high value, would `0` as an actual non-public org id be invalid?